core/src/test/scala/unit/kafka/cluster/PartitionLockTest.scala

```diff
@@ -283,7 +283,7 @@ class PartitionLockTest extends Logging {
         val log = super.createLog(isNew, isFutureReplica, offsetCheckpoints, None)
         val logDirFailureChannel = new LogDirFailureChannel(1)
         val segments = new LogSegments(log.topicPartition)
-        val leaderEpochCache = Log.maybeCreateLeaderEpochCache(log.dir, log.topicPartition, logDirFailureChannel, log.config.messageFormatVersion.recordVersion)
+        val leaderEpochCache = Log.maybeCreateLeaderEpochCache(log.dir, log.topicPartition, logDirFailureChannel, log.config.messageFormatVersion.recordVersion, "")
```

Comment:
As above mentioned, if the default logPrefix is empty string, then the following change in tests are not needed.

Reply:
Explained here: https://github.com/apache/kafka/pull/10742#discussion_r637344863